server/src/concepts/user/useCases/signInUser.js

```diff
@@ -0,0 +1,26 @@
+import User from "../model/User.js";
+import bcrypt from "bcrypt";
+import { generateToken } from "../../authorization/controllers/auth.js";
+
+export const signInUserFunc = async (email, password) => {
+  if (!email || !password) {
```

Comment:
Maybe it would be better to check if there is no email or password provided in controllers, we could there already send res.status(400)